dashboard/app/models/pd/enrollment_notification.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: pd_enrollment_notifications
+#
+#  id               :integer          not null, primary key
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  pd_enrollment_id :integer          not null
+#  name             :string(255)
+#
+# Indexes
+#
+#  index_pd_enrollment_notifications_on_pd_enrollment_id  (pd_enrollment_id)
+#
+
+class Pd::EnrollmentNotification < ActiveRecord::Base
+  belongs_to :enrollment, class_name: 'Pd::Enrollment', foreign_key: :pd_enrollment_id
+end
```

Comment:
help me understand rails better, why is this line not just `belongs_to :pd_enrollment` ?